core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -606,6 +606,33 @@ public void resetState() {
     mPathProperties.resetState();
   }
 
+  @Override
+  public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMap) {
+    Map<String, Boolean> result = new HashMap<>();
+    int successCount = 0;
+    for (Map.Entry<String, String> entry : propertiesMap.entrySet()) {
+      try {
+        PropertyKey key = PropertyKey.fromString(entry.getKey());
+        if (key.isDynamic()) {
+          String oldValue = ServerConfiguration.get(key);
+          ServerConfiguration.set(key, entry.getValue(), Source.RUNTIME);
+          result.put(entry.getKey(), true);
+          successCount++;
+          LOG.debug("Update property {} from {} to {}",
+              key.getName(), oldValue, ServerConfiguration.get(key));
+        } else {
+          LOG.debug("Update a non-dynamic property {} is banned", key.getName());
+          result.put(entry.getKey(), false);
+        }
+      } catch (Exception e) {
+        result.put(entry.getKey(), false);
+        LOG.error("Update property {} to {} met exception", e);
```

Comment:
unbalanced arguments

Reply:
Good catch, thank you to show me this error, I just want to display the key and value.